E/Econometrics/Compat.toml

```diff
@@ -1,12 +1,19 @@
 [0]
-FillArrays = "0.6"
-Optim = "0.18"
 Parameters = "0.10"
-StatsBase = "0.31"
 StatsModels = "0.6"
 
 ["0-0.2.0"]
 julia = "1.1.0-1"
 
+["0-0.2.2"]
+FillArrays = "0.6"
```

Comment:
These should probably be retroactively upper bounded.

Reply:
Technically the previous versions should work with `0.6, 0.7` and potentially any going forward for now.